power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/linear_pf_solver.hpp

```diff
@@ -113,13 +115,16 @@ template <bool sym> class LinearPFSolver {
     }
 
     void calculate_result(YBus<sym> const& y_bus, PowerFlowInput<sym> const& input, MathOutput<sym>& output) {
-        common_solver_functions::calculate_result(y_bus, input, *sources_per_bus_, *load_gens_per_bus_, output,
-                                                  [](Idx /*i*/) { return LoadGenType::const_y; });
+        detail::calculate_result(y_bus, input, *sources_per_bus_, *load_gens_per_bus_, output,
+                                 [](Idx /*i*/) { return LoadGenType::const_y; });
     }
 };
 
 template class LinearPFSolver<true>;
 template class LinearPFSolver<false>;
+} // namespace math_solver
+
+template <bool sym> using LinearPFSolver = math_solver::LinearPFSolver<sym>;
```

Comment:
for this one and the others: do you think it's a good idea to expose these not to `namespace power_grid_model` but only to `namespace power_grid_model::math_solver`? the `MathSolver` itself does have unrestricted access to anything in the `namespace power_grid_model::math_solver`, so there should be no issue there

Reply:
I changed it so that each solver has its own namespace now

Reply:
ok but it should be the other way around:

all `using` declarations should be in the `namespace power_grid_model::math_solver` namespace, not the parent `namespace power_grid_model`